mongoDb/index.js

```diff
@@ -0,0 +1,73 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/tetraflix');
+
+const db = mongoose.connection;
+
+db.on('error', () => {
+  console.error('connection error');
+});
+
+db.once('open', () => {
+  console.log('Now connected to tetraflix database');
+});
+
+// schema for movies
```

Comment:
I think you can remove this comment, since your variable naming is pretty clear (which is better than needing comments 👍 ).

Reply:
done